frontend/src/v5/store/tickets/tickets.selectors.ts

```diff
@@ -79,8 +80,8 @@ export const selectTickets = createSelector(
 	selectTicketsRaw,
 	selectTicketsGroups,
 	(state, modelId) => modelId,
-	(state) => state,
-	(ticketsList, groups, modelId, storeState): ITicket[] => {
+	(ticketsList, groups, modelId): ITicket[] => {
+		const storeState = getState();
```

Comment:
Why has this been changed to use `getState()`?

Reply:
Because of the way selectors cache data. In order for them to avoid recomputing the output, they check the current parameters against the previous ones. When you pass the `storeState` as we did before, any changes, even the ones not affecting tickets, was causing this to re-compute the data, which impacted negatively the performance